mod/state-transition/pkg/core/state_processor_staking.go

```diff
@@ -203,25 +208,31 @@ func (sp *StateProcessor[
 		return err
 	}
 	numWithdrawals := len(expectedWithdrawals)
+	if sp.cs.DepositEth1ChainID() == spec.BoonetEth1ChainID &&
+		slot >= spec.BoonetFork2Height {
+		// Enforce that there is at least one withdrawal for EVM inflation.
+		if numWithdrawals == 0 {
+			return ErrZeroWithdrawals
+		}
```

Comment:
This is an extra check on a state that we control. I don't believe it is necessary, but I am fine keeping this as is, at least until we have properly validated all genesis and chain specs.

Reply:
No wait, this is not useless. We should require that we have at least one withdrawal not only on Boonet post fork2 but also in any network that is not Bartio and Boonet pre fork 2 (like the starting with main and ending with net).
Also for this network, we need block building logic to duly include the EVM inflation withdrawal, which I do not think we currently do.